indico/modules/events/ical.py

```diff
@@ -145,6 +146,17 @@ def generate_event_component(
     if data['description']:
         component['description'] = data['description']
 
+    # If the user exists and the add_icloud_alerts preference isn't blank
+    if user and user.settings.get('add_ical_alerts', '') != '':
+        alarm = icalendar.Alarm()
+        alarm.add('prodid', '-//CERN//INDICO//EN')
```

Comment:
is this needed? at least in one of our plugins we don't add it and it seems to work

Reply:
That was just cargo-culting some other code I saw nearby. Not sure if it's needed or not

Reply:
removed